editor/editor_help.cpp

```diff
@@ -4573,8 +4573,6 @@ EditorHelpBitTooltip::EditorHelpBitTooltip(Control *p_target) {
 
 	p_target->connect(SceneStringName(mouse_exited), callable_mp(this, &EditorHelpBitTooltip::_start_timer));
 	p_target->connect(SceneStringName(gui_input), callable_mp(this, &EditorHelpBitTooltip::_target_gui_input));
-
-	set_process_internal(true);
```

Comment:
This is not a correct solution, but simply disabling the functionality. This will make the tooltips inconvenient to use.

Reply:
Yeah, really sorry about that, for some reason I thought this internal process was not being used, and I didn't see anything break so I thought it was okay. I've analyzed the problem properly, I'll push the correct change and make comment about it.